Tests/UnitTests/Core/Detector/ProtoBinningTests.cpp

```diff
@@ -0,0 +1,63 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <boost/test/data/test_case.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "Acts/Detector/ProtoBinning.hpp"
+
+using namespace Acts::Experimental;
+
+BOOST_AUTO_TEST_SUITE(Detector)
+
+BOOST_AUTO_TEST_CASE(ProtoBinningEquidistant) {
+  // An invalid binning
+  BOOST_CHECK_THROW(
+      ProtoBinning(Acts::binX, Acts::detail::AxisBoundaryType::Bound, 15., 20.,
+                   0),
+      std::invalid_argument);
+
+  // Another invalid binning
+  BOOST_CHECK_THROW(
+      ProtoBinning(Acts::binX, Acts::detail::AxisBoundaryType::Bound, 150., 20.,
+                   0),
+      std::invalid_argument);
```

Comment:
```suggestion
      ProtoBinning(Acts::binX, Acts::detail::AxisBoundaryType::Bound, 150., 20.,
                   1u),
      std::invalid_argument);
```
Maybe we could set here a valid `nbins`, to make sure that right conditions gets checked.

Reply:
Yup, fixed.